Cargo.toml

```diff
@@ -18,17 +18,17 @@ backend-mmap = []
 backend-atomic = ["arc-swap"]
 
 [dependencies]
-libc = ">=0.2.39"
-arc-swap = { version = ">=1.0.0", optional = true }
+libc = "0.2.39"
+arc-swap = { version = "1.0.0", optional = true }
 
 [target.'cfg(windows)'.dependencies.winapi]
-version = ">=0.3"
+version = "0.3"
 features = ["errhandlingapi", "sysinfoapi"]
 
 [dev-dependencies]
 criterion = "0.3.0"
-matches = ">=0"
-vmm-sys-util = ">= 0.4.0"
+matches = "0"
```

Comment:
This dependency will be updated up until < 1.0. If we want to only update as specified in the commit message, we should add this as `0.1` instead.

Reply:
Good catch! Fixed it.